ga/20.0.0.8/oidcProvider/Dockerfile.java11

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:kernel-java11
+FROM websphere-liberty:20.0.0.8-kernel-java11
```

Comment:
Not really sure what this was supposed to point to before, the `kernel-java11` image hasn't been updated in 9 months. I think either leave as `kernel-java11` or change to `ibmcom/websphere-liberty:kernel-java11-openj9-ubi`